### PR TITLE
Add auto-generated issue template

### PR DESCRIPTION
**Pull request by stuartmccoll**
This commit adds an issue template auto-generated by the GitHub 'Set up templates' functionality.

More information on this can be found at https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
```
Comment (gidsg) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: Can we add a header here for _Actual behaviour_ and maybe remove the step 4 from to reproduce.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
```
Comment (gidsg) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]: Not sure what this is adding unless it auto populates? Maybe just delete this block?

Comment (stuartmccoll) [reply .github/ISSUE_TEMPLATE/bug_report.md:8]: Used for auto population 👍.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (stuartmccoll) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: 👍

**Pull request merged**